dhis-2/dhis-api/src/main/java/org/hisp/dhis/resourcetable/ResourceTableStore.java

```diff
@@ -63,4 +63,11 @@
      * @param batchArgs the arguments to use for the update statement.
      */
     void batchUpdate( int columns, String tableName, List<Object[]> batchArgs );
+
+    /**
+     * Returns all data years available in the system (both from aggregate and
+     * tracker) with 5 years previous and future additions.
+     * [extra_5_previous_years, data_years, extra_5_future_year]
+     */
+    List<Integer> getAvailableDataYears();
```

Comment:
Just curious what the difference between `generateDataYears` and `getAvailableDataYears` are, as they have the same Javadoc?